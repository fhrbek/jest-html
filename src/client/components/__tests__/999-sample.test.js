/* eslint-env jest */
/* eslint-disable global-require, import/newline-after-import */
import React from 'react';
import renderer from 'react-test-renderer';

// https://github.com/facebook/react/issues/7386#issuecomment-238091398
jest.mock('react-dom');

const Component = () => (
  <div style={{}}>
    <b>Hello</b> there
  </div>
);

describe('Sample', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Component />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
