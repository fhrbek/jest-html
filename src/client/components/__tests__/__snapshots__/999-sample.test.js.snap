// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample renders correctly 1`] = `
<div
  style={Object {}}
>
  <b>
    Hello
  </b>
   there
</div>
------------HTML PREVIEW---------------
<div style="">
  <b>
    Hello
  </b>
   there
</div>
`;
